tests/unittests/test_client_utilities.py

```diff
@@ -154,46 +162,73 @@ def test_timeout_error_get_retry(self, mocked_time_sleep, mocked_session_post, m
 
     def test_503_backend_triggers_no_retry(self, mocked_time_sleep, mocked_session_post, mocked_session_request, mocked_request_post):
         client = Client(self.config, self.config_path)
-        with self.assertRaisesRegex(requests.exceptions.RequestException, 'Raise for status'):
+        with self.assertRaises(GoogleAnalyticsClient.GoogleAnalyticsBackendError):
             client.post("503-BACKENDERROR")
 
         # Assert we gave up only after the first try
         self.assertEqual(mocked_session_post.call_count, 2)
```

Comment:
Why does it call for 2 times instead of 1?

Reply:
One for the actual call and the other for post call in ensure_access_token()